resident.rb

```diff
@@ -0,0 +1,97 @@
+class Raspberry 
+	attr_reader :@@states = ["absence", "bloom", "green", "red"]
```

Comment:
This is the internal states of the berry, why does it have a reader?

Reply:
I will remove attr_reader.